packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/arguments/Action.java

```diff
@@ -0,0 +1,93 @@
+package expo.modules.imagemanipulator.arguments;
+
+import android.support.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Action extends HashMap<String, Object> {
```

Comment:
Does it need to be a subclass of `Map`? It looks like it holds all the information by itself… 🤔

Reply:
Yeap, bacause I do not implement `size()` method here.